remoting/src/main/java/org/apache/rocketmq/remoting/transport/mqtt/MqttRemotingClient.java

```diff
@@ -0,0 +1,307 @@
+                    if (nettyClientConfig.isUseTLS()) {
+                        if (null != sslContext) {
+                            pipeline.addFirst(defaultEventExecutorGroup, "sslHandler", sslContext.newHandler(ch.alloc()));
+                            log.info("Prepend SSL handler");
+                        } else {
+                            log.warn("Connections are insecure as SSLContext is null!");
+                        }
+                    }
+                    pipeline.addLast(
+                        defaultEventExecutorGroup,
+                        MqttEncoder.INSTANCE,
+                        new MqttDecoder(),
+                        new IdleStateHandler(0, 0, nettyClientConfig.getClientChannelMaxIdleTimeSeconds()),
+                        new NettyConnectManageHandler(),
+                        new NettyClientHandler());
+                }
+            });
+        startUpHouseKeepingService();
+    }
+
+    @Override
+    public void shutdown() {
+        super.shutdown();
+        try {
+            clearChannels();
+            if (this.eventLoopGroupWorker != null) {
+                this.eventLoopGroupWorker.shutdownGracefully();
+            }
+            if (this.defaultEventExecutorGroup != null) {
+                this.defaultEventExecutorGroup.shutdownGracefully();
+            }
+            if (this.publicExecutor != null) {
+                this.publicExecutor.shutdown();
+            }
+        } catch (Exception e) {
+            log.error("NettyRemotingClient shutdown exception, ", e);
+        }
+    }
+
+    @Override
+    public void registerInterceptorGroup(InterceptorGroup interceptorGroup) {
+        this.interceptorGroup = interceptorGroup;
+    }
+
+    @Override
+    public void updateNameServerAddressList(List<String> addrs) {
+        super.updateNameServerAddressList(addrs);
+    }
+
+    @Override
+    public RemotingCommand invokeSync(String addr, final RemotingCommand request, long timeoutMillis)
+        throws InterruptedException, RemotingConnectException, RemotingSendRequestException, RemotingTimeoutException {
+        long beginStartTime = System.currentTimeMillis();
+        final RemotingChannel remotingChannel = this.getAndCreateChannel(addr, timeoutMillis);
+        if (remotingChannel != null && remotingChannel.isActive()) {
+            try {
+                long costTime = System.currentTimeMillis() - beginStartTime;
+                if (timeoutMillis < costTime) {
+                    throw new RemotingTimeoutException("invokeSync call timeout");
+                }
+                RemotingCommand response = this.invokeSyncWithInterceptor(remotingChannel, request, timeoutMillis - costTime);
+                return response;
+            } catch (RemotingException remotingException) {
+                if (remotingException instanceof RemotingSendRequestException) {
+                    log.warn("invokeSync: send request exception, so close the channel[{}]", addr);
```

Comment:
Please start with a capital letter in the comment.

Reply:
@duhenglucky modified according to comment. Thanks for that.